src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -178,9 +210,6 @@ void SocketConnection::serialize(jalib::JBinarySerializer& o)
   TcpConnection::TcpConnection(int domain, int type, int protocol)
   : Connection(TCP_CREATED)
   , SocketConnection(domain, type, protocol)
```

Comment:
I don't understand why we can delete things like `_bindAddrlen`.  The variable is used lower in the code.  Don't we still need to set a default value for these variables?